examples/layers/preprocessing/random_flip_demo.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+   random_flip_demo.py shows how to use the RandomFlip preprocessing layer
+   for object detection. An image is downloaded from URL, both the image and
+   bounding boxes are augmented and displayed using matplotlib.
+"""
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+import requests
+import tensorflow as tf
+from PIL import Image
+
+from keras_cv.layers import preprocessing
```

Comment:
from keras_cv import layers

Prefer top level symbols.

Instead of:
```
from tf.keras.layers import preprocessing
```

Prefer:
```
from tf.keras import layers
```

Reply:
okay